server/src/main/java/org/spine3/server/event/enrich/EnrichmentFunction.java

```diff
@@ -111,4 +137,16 @@ public String toString() {
                           .add("enrichmentClass", enrichmentClass)
                           .toString();
     }
+
+    /**
+     * Checks whether this instance of {@code EnrichmentFunction} is active.
+     *
+     * <p>Throws {@link IllegalStateException} if the instance is not active.
+     */
+    protected void ensureActive() {
+        if(!isActive()) {
+            throw new IllegalStateException("The given instance of " + getClass() + " is not active at the moment. " +
+                                                    "Please use `#activate()` first.");
```

Comment:
Is it a set practice to reference methods via the hash sign in the console output? I though it applies only to Javadocs (and this is so because they follow HTML anchors).